sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -344,6 +344,17 @@ export interface CloudEvent<T> {
     type: string;
 }
 
+// @public
+export interface CloudEventAdapterParameters {
+    dataschema?: string;
```

Comment:
`dataSchema`? Or is this an unfortunate external standard?

Reply:
It appears that this particular field is not coming from the cloud event spec but I noticed that all fields in the [`SendCloudEventInput<T>`](https://github.com/Azure/azure-sdk-for-js/blob/8c09c32d3292dcdc91b0a78e4dc072cf51ced36d/sdk/eventgrid/eventgrid/src/models.ts#L76) follow the same all-lower-case pattern. /cc @ellismg

Reply:
https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#dataschema